charts/opentelemetry-kube-stack/values.yaml

```
@@ -510,14 +510,22 @@ collectors:
             http:
               endpoint: 0.0.0.0:4318
       processors:
+        resourcedetection/env:
+          detectors: [env, k8snode]
+          timeout: 2s
+          override: false
+        transform/set-hostname:
+          error_mode: ignore
+          trace_statements:
+            - set(resource.attributes["host.name"], resource.attributes["k8s.node.name"]) where resource.attributes["host.name"] == nil
+          metric_statements:
+            - set(resource.attributes["host.name"], resource.attributes["k8s.node.name"]) where resource.attributes["host.name"] == nil
+          log_statements:
+            - set(resource.attributes["host.name"], resource.attributes["k8s.node.name"]) where resource.attributes["host.name"] == nil
```

What do you think if we use the [resource](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourceprocessor) processor instead? That will prevent having to check the existence of the `host.name` and `k8s.node.name` resource attributes (and the configuration will be the same for any signal).

```suggestion
        resource/hostname:
          attributes:
            - key: host.name
              from_attribute: k8s.node.name
              action: insert
```

---

Thanks for the suggestion, please review updated code.